windows/vhba_hot_add_remove/vhba_test.yml

```diff
@@ -6,6 +6,36 @@
 # - lsilogic: no inbox driver in Windows guest OS now, so no testing.
 # - buslogic: not supported in 64bit Windows guest OS, so no testing.
 #
+- name: "Set fact of whether VM NVMe version 1.3 is enabled by default"
+  when:
+    - vm_hardware_version_num | int >= 21
+    - test_disk_ctrl_type == 'nvme'
+    - vm_nvme_spec13_enabled is undefined
+  block:
+    - name: "For Windows {{ guest_os_product_type }} with guest ID {{ vm_guest_id }}"
+      ansible.builtin.set_fact:
+        vm_nvme_spec13_enabled: "{{ vm_guest_id | regex_search('windows(\\d+)srvNext', '\\1') | first | int >= 2019 }}"
```

Comment:
This task will fail if `vm_guest_id` is `windows2019srv_64Guest`. It's better to add a condition in `when` to check `vm_guest_id` is match `windows(\\d+)srvNext`

Reply:
Yeah, right, thanks for catching this.